.github/workflows/trivy_periodic_image_scan.yml

```diff
@@ -0,0 +1,29 @@
+---
+#
+# This workflow scans the published container images
+# for new vulnerabilities daily, publishing findings.
+# Findings will be associated with the 'main' branch
+# of the repo' in the GitHub Security tab.
+#
+name: Trivy Periodic Image Scan
+
+on:
+  schedule:
+    # run daily
+    - cron: "0 0 * * *"
+
+jobs:
+  trivy-matrix:
```

Comment:
how do we respond to failed scans?  will someone get notified?

Reply:
We would treat Trivy findings like Dependabot findings:  Once a month we have a look and open a Jira issue if there is a finding to address.

Reply:
ok